src/components/Accordion/Accordion/__stories__/accordion.stories.mdx

```diff
@@ -0,0 +1,149 @@
+import { createComponentTemplate } from "../../../../storybook/functions/create-component-story";
+import Checkbox from "../../../Checkbox/Checkbox.js";
+import { WIZARD, BREADCRUBMS, STEPPER } from "../../../../storybook/components/related-components/component-description-map";
+import "./accordion.stories.scss";
+
+<Meta
+  title="Components/Accordion"
+  component={ Accordion }
+/>
+
+<!--- Component template -->
+
+export const accordionTemplate = (args) => {
+  return (
+    <div className="monday-storybook-accordion_small-wrapepr">
+      <Accordion defaultIndex={[1]}>
+        <AccordionItem title="Notifications" />
+        <AccordionItem title="Setting" />
+        <AccordionItem title="Info" />
+        <AccordionItem title="Profile" />
+        <AccordionItem title="Permisions" />
+        <AccordionItem title="Seurity" />
+        <AccordionItem title="Connectivity" />
+        <AccordionItem title="Integration" />
+        <AccordionItem title="Assets" />
+      </Accordion>
+    </div>
+  )
+};
+
+<!--- Component documentation -->
+
+# Accordion
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Accordion is a vertically stacked list of items. Each item can be "expanded" or "collapsed" to reveal the content within with that item.
+
+<Canvas>
+  <Story name="Overview"
+         args={ {} }>
+    { accordionTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Accordion } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Use accordion reduce clutter and chunk the information one by one",
+  "Accordion Label must be short, clear, and understandable to indicate what's inside",
+  "Default state of accordion is closed unless used for navigation",
+  "Accordion content can include icons, radio buttons, and checkboxes.",
+]}/>
+
+## Variants
+### Multi active
+Each section can be expanded without closing the others
+<Canvas>
+  <Story name="Multi active">
+    <div className="monday-storybook-accordion_small-wrapepr">
+      <Accordion allowMultiple defaultIndex={[1, 3]}>
+        <AccordionItem title="Notifications"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Setting"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Connectivity"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Integration"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+        <AccordionItem title="Assets"><div className="monday-storybook-accordion_small-box" /></AccordionItem>
+      </Accordion>
+    </div>
+  </Story>
+</Canvas>
+
+### Single active
+Only one section can be open at the time
+<Canvas>
+  <Story name="Single active">
```

Comment:
please set the accordion width by using the className prop instead of by using a wrapper